modules/services/picom.nix

```diff
@@ -269,6 +271,21 @@ in {
       '';
     };
 
+    extraWintypes = mkOption {
```

Comment:
Please add a test to verify that this generates the expected config.

Reply:
I'm not very sure how to do that :sweat_smile:. Could you link to an example where there's a test for a similar option? (Also since its just a string that's added directly to the config, why is a test required?)

Reply:
You can basically look at any of the tests in the `tests` directory. Really, the entire module should really have a test, but it seems like none was added initially.